### DISCUSSION TITLE
Disable delay when holding movement keys

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [helix-editor/helix#9187] [Troubleshooting]
Is there a way to disable the delay. When I hold down a movement key the cursor dosen't instantly start to move in the direction but there is a delay

**Comment by dead10ck** [comment]
This sounds like an issue outside of Helix. There should not be a delay.

**Comment by Vilhelm-Ian** [comment]
The delay is not present in neovim.

**Comment by the-mikedavis** [comment]
Are you using tmux?

**Comment by Vilhelm-Ian** [comment]
konsole

**Comment by Vilhelm-Ian** [comment]
I just tested it out again. The delay happens in neovim also. So the issue is not with helix
